backend-axum/src/main.rs

```diff
@@ -3,22 +3,40 @@ use axum::{
     extract::{FromRef, FromRequestParts},
     http::{request::Parts, StatusCode},
     routing::{delete, get, post, put},
-    Extension, Router,
+    Router,
+};
+use tower::ServiceBuilder;
+
+use tower_http::{
+    cors::CorsLayer,
+    trace::TraceLayer,
+    add_extension::AddExtensionLayer,
 };
 use sqlx::postgres::{PgPool, PgPoolOptions};
 use std::{net::SocketAddr, time::Duration};
-// use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+use tracing_subscriber::EnvFilter;
 
 mod configuration;
 mod controllers;
 mod errors;
 mod models;
+mod middleware;
 
 #[tokio::main]
 async fn main() {
+
+    // RUST_LOG=backend_axum=info
+    // either in .env or before cargo run
+    dotenvy::dotenv().ok();
```

Comment:
@NeuroPyPy This is already happening, this is what the configuration module is for.

Reply:
yea I was just testing something and forgot to remove that